cypress/integration/crud.spec.js

```diff
@@ -1,36 +1,16 @@
-describe('Notes', () => {
-  beforeEach(() => {
-    cy.visit('http://notes-serverless-app.com')
-
-    cy.get('.navbar-nav a:contains(Login)').click()
-
-    cy.get('#email').type(Cypress.env('user'))
-    cy.get('#password').type(Cypress.env('password'))
-    cy.get('button[type="submit"]').click()
-  })
+/// <reference types="Cypress" />
 
-  it('creates a note', () => {
-    cy.contains('Create a new note').click()
-
-    cy.get('#content').type('My note')
-    cy.contains('Create').click()
-
-    cy.get('.list-group').should('contain', 'My note')
-  })
-
-  it('edits a note', () => {
-    cy.get('.list-group').contains('My note').click()
-    cy.get('#content').type(' updated')
-    cy.contains('Save').click()
-
-    cy.get('.list-group').should('contain', 'My note updated')
-    cy.get('.list-group:contains(My note updated)').should('be.visible')
-  })
-
-  it('deletes a note', () => {
-    cy.get('.list-group').contains('My note updated').click()
-    cy.contains('Delete').click()
-
-    cy.get('.list-group:contains(My note updated)').should('not.exist')
+describe('Notes', () => {
+  beforeEach(() => cy.login())
+  
+  it('Creates, edits and deletes a note', () => {    
+    const data = {
+      content: 'My note',
+      newContent: 'My note updated'
+    }
+    
+    cy.createsANote(data)
```

Comment:
Ideally, expectations should be explicit on the test body to understand exactly what's the behavior we're testing.

Reply:
Done!